src/chocolatey/infrastructure.app/builders/ConfigurationBuilder.cs

```diff
@@ -438,7 +438,7 @@ private static void SetGlobalOptions(IList<string> args, ChocolateyConfiguration
                             "Proxy User Name - Explicit proxy user (optional). Requires explicit proxy (`--proxy` or config setting). Overrides the default proxy user of '{0}'.".FormatWith(config.Proxy.User),
                             option => config.Proxy.User = option.UnquoteSafe())
                         .Add("proxy-password=",
-                            "Proxy Password - Explicit proxy password (optional) to be used with username. Requires explicit proxy (`--proxy` or config setting) and user name.  Overrides the default proxy password (encrypted in settings if set).",
+                            "Proxy Password - Explicit proxy password (optional) to be used with user name. Requires explicit proxy (`--proxy` or config setting) and user name (`--proxy-user` or config setting).  Overrides the default proxy password (encrypted in settings if set).",

```

Comment:
```suggestion
                            "Proxy Password - Explicit proxy password (optional) to be used with user name. Encrypted. Requires explicit proxy (`--proxy` or config setting) and user name (`--proxy-user` or config setting).  Overrides the default proxy password.",
```

I think it's a good idea to be consistent here with the wording of the other settings above when mentioning whether a given setting is encrypted at rest or not.

Reply:
That makes sense to me. I've made the change and amended it into the commit. I've also rebased onto latest develop and force pushed.